### PR TITLE
 reverted week 1 assessment

### PR DESCRIPTION
**Pull request by henrike32**
# Submitting Week 1 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [x] Coding Challenges
- [x] Coding Concepts
- [x] Interview Questions

### 📝 Feedback

If you would like specific feedback on a particular problem please leave details here:
I think I messed up trying to turn in the assessment.

### CODE REVIEW

File: code-challenges.js [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+// const tempStatus = (temperature1,boilingPoint) => {
+//     if (temperature1 > boilingPoint) {
+//     return " above boiling point"
+//     } else if (temperature1 === boilingPoint) {
+//       return  " at boiling point"
+//     } else {
+//       return  " below boiling point"
+//     }
+// }
```
Comment (chrisyhlee) [inline code-challenges.js:79]: Remember that the purpose of functions and parameters is to make the function reusable for as many times as we want to use it. When you create parameters, they are placeholders for the arguments, which are given when you actually call the function. For example, when you `console.log(tempStatus(temperature1, boilingPoint))`, you are performing a function call on `tempState()` and giving it the arguments of `temperature1` and `boilingPoint`. Keep this in mind when you create your functions and parameters. You should only need to write out 1 function, and then you can use that function for every variable that the question gives you!

File: code-challenges.js [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4];
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3];
+
+// const wComb = [padres1984WorldSeriesRuns , padres1998WorldSeriesRuns];
+// const combLength = [wComb.length];
+// console.log(combLength);
+// Expected output: 9
+// Op: [2] was the output I got, I tried to modify the code and still can't make it work, I will inquire about this challenge to one of the instructors whenever I get the chance.
+// Commented the code for the next challenge.
```
Comment (chrisyhlee) [inline code-challenges.js:141]: Here, you are storing the variables inside an array -- so you have an array that is holding 2 arrays. When you run .length on `wComb`, it is counting the elements inside (in this case, 2 arrays), so it is returning 2. Think about what methods are available to you to combine arrays together!

File: code-challenges.js [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+// Process:
+// Create a const named `revCohort`.
+// Use the .split method to convert the string into an array of characters, use .reverse method to reverse 
+// the order of the characteres in the string and then use the .join method to convert the reversed array back to a string.
+//
+// const currentCohort = "Charlie 2023";
+// const revCohort = currentCohort.split("").reverse().join("");
+// console.log(revCohort);
+// Expected output: "3202 eilrahC".
+// op : "3202 eilrahC".
```
Comment (chrisyhlee) [inline code-challenges.js:158]: Nice work!

File: code-challenges.js [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+
+// ----4) Create the code that will return the last index of the given value from the array using the test variables provided below.
+// Pseudo code:
+// Quick Note >>>>>>>> To make things easier , input will be called `Inp`,output as `Op` and `console.log` as `log`. 
+// Inp: New const.
+// Op: Last index number of the given value in the array.
+// Process:
+// Create a cons named `theValue`.
+// Use the method .lastIndexOf.
+// Log the cons created.
```
Comment (chrisyhlee) [inline code-challenges.js:169]: I see you referring to variables as "cons" in your pseudo code. Make sure you are referring to them as variables, not as their variable declaration keywords.

File: code-challenges.js [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+// Op: array sorted from largest to smallest.
+// Process:
+// Use the sort method to sort the numbers in the array.
+// Find the right function to make the sort method function properly.
+// Log sanDiegoSummerTemperatures.
+// const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
+// sanDiegoSummerTemperatures.sort((a,b) => b - a);
+// console.log(sanDiegoSummerTemperatures);
+// Expected output: [82, 80, 79, 77, 76, 73, 72]
+// Op:[82, 80, 79, 77, 76, 73, 72]
```
Comment (chrisyhlee) [inline code-challenges.js:200]: Good research on `.sort()`

File: coding-concepts.js [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+const languages = ["JavaScript", "Ruby", "Python", "C++"]
+const index = 1
+console.log(languages[index])
+
+// a) Your answer: JavaScript . 
+// b) Verify and explain: Ruby is the ouput . In my mind I thought I counted the 0 1 in the index,
+// forgot to count the 0 to the right index. You have 2 variables, and the variable languages is an array ,
+// the variable index is a number. You have a log that is asking you what is the index of the other variable 
+// using a variable to give you the number of the index . // observation: Hopefully with time I can express
+// myself better when explaining the work done.
```
Comment (chrisyhlee) [inline coding-concepts.js:51]: Good correction!

File: coding-concepts.js [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+// --------------------4) What will this log?
+
+// const weekendDays = ["saturday", "sunday"]
+// console.log(weekendDays.toUpperCase())
+
+// a) Your answer: "SATURDAY, "SUNDAY"
+// b) Verify and explain: TypeError: weekendDays.toUpperCase is not a function //
+// The `toUpperCase()` function is a method that can be called on strings, but `weekendDays`
+// is an array. I don't know exactly how to do the capitalization of both strings, maybe make an
+//  iteration?
```
Comment (chrisyhlee) [inline coding-concepts.js:62]: Nice work researching why it didn't work as expected

**Review by chrisyhlee — APPROVED** [review]
Nice work on the week 1 assessment! This is currently showing Novice level. Make sure to review the comments I left, and if you have any questions, you can always reach out to me on slack! I would suggest you revisit functions and parameters. Remember that you want your functions to be reusable, which means they should work not just for the test variables. Also, remember that parameters are placeholders for the arguments! With this in mind, revisit some of the challenges and see if you can refactor your code to make your function dynamic!

**Comment by henrike32** [comment]
Good morning Chris,
Thank you for the feedback,
I will do as you suggested,

**Pull request merged**